quill-spark/src/main/scala/io/getquill/QuillSparkContext.scala

```diff
@@ -0,0 +1,76 @@
+package io.getquill
+
+import scala.util.Success
+import scala.util.Try
+import org.apache.spark.sql.Dataset
+import org.apache.spark.sql.{ Encoder => SparkEncoder }
+import org.apache.spark.sql.SQLContext
+import io.getquill.context.Context
+import org.apache.spark.rdd.RDD
+import language.implicitConversions
+import io.getquill.context.spark.Encoders
+import io.getquill.context.spark.Decoders
+import io.getquill.context.spark.SparkDialect
+import io.getquill.context.spark.Binding
+import io.getquill.context.spark.DatasetBinding
+import io.getquill.context.spark.ValueBinding
+import scala.reflect.ClassTag
+
+object QuillSparkContext extends QuillSparkContext
+
+trait QuillSparkContext
+  extends Context[SparkDialect, Literal]
+  with Encoders
+  with Decoders {
+
+  type Result[T] = Dataset[T]
+  type RunQuerySingleResult[T] = T
+  type RunQueryResult[T] = T
+
+  def close() = {}
+
+  def probe(statement: String): Try[_] = Success(Unit)
+
+  val idiom = SparkDialect
+  val naming = Literal
```

Comment:
Does it mean that using different naming strategies is not possible?

Reply:
Yes, given that the new module uses spark encoders and they don't support naming strategies, only `Literal` can be used.